include/seqan3/std/algorithm

```diff
@@ -37,80 +39,4 @@
 #include <range/v3/algorithm/sort.hpp>
 #include <range/v3/algorithm/transform.hpp>
 
-#include <seqan3/core/platform.hpp>
-
-namespace std::ranges
-{
-
-/*!\typedef std::ranges::all_of
- * \brief Alias for ranges::all_of. Checks if unary predicate `p` returns `true` for all elements in the range.
- */
-using SEQAN3_DOXYGEN_ONLY(all_of =) ::ranges::all_of;
```

Comment:
I don't understand this change :see_no_evil: 
Where do you alias the range algorithms now?
Why are all the range includes still in there if not used?

Reply:
> Where do you alias the range algorithms now?

The entire namespace is imported.

> Why are all the range includes still in there if not used?

They are used :smile: